README.md

```diff
@@ -14,12 +14,12 @@ Once you have finished the home page using flex box module, go build the service
 
 ## Objectives
 
-* [ ] Home Page Refactor:
+* [X] Home Page Refactor:
```

Comment:
first time seeing some one keep track of their progress this way, Good way to stay organized.